tests/acceptance/mocks/mock_dbus_server.py

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/python
+# Copyright 2021 Northern.tech AS
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+
+from gi.repository import GLib
+from pydbus import SystemBus
+from pydbus.generic import signal
+
+
+class IoMenderAuthenticationIface(object):
+    """
+<node>
+	<interface name="io.mender.Authentication1">
+		<method name="GetJwtToken">
+			<arg type="s" name="token" direction="out"/>
+			<arg type="s" name="server_url" direction="out"/>
+		</method>
+		<method name="FetchJwtToken">
+			<arg type="b" name="success" direction="out"/>
+		</method>
+        <signal name="JwtTokenStateChange">
+			<arg type="s" name="token"/>
+			<arg type="s" name="server_url"/>
+		</signal>
+		<method name="MockSetJwtToken">
+			<arg type="s" name="token" direction="in"/>
+			<arg type="s" name="server_url" direction="in"/>
+		</method>
+		<method name="MockSetJwtTokenAndEmitSignal">
+			<arg type="s" name="token" direction="in"/>
+			<arg type="s" name="server_url" direction="in"/>
+		</method>
+	</interface>
+</node>
+	"""
+
+    token = ""
```

Comment:
Nit:

I don't actually see these used anywhere :thinking: Are they meant to be object local?

Reply:
They are class attributes at the beginning, overridden later by object attributes. In theory, a user could call `GetJwtToken` before the test sets `self.token` and `self.server_url` via `MockSetJwtToken`.

Reply:
Why not just use `__init__`?

Reply:
Yes, of course! I don't know what I was thinking :weary:

Reply:
Fixed now. Thanks!